spec/shortcuts_spec.rb

```diff
@@ -142,6 +142,14 @@
       m2 = NMatrix.random(2,:seed => 62)
       m3 = NMatrix.random(2,:seed => 65)
 
+      expect(m1.stype).to eq(:dense)
+      expect(m1.dtype).to eq(:float64)
+      
+      expect(m2.stype).to eq(:dense)
```

Comment:
Hi @mohawkjohn 
I deleted it because I thought it was redundant. I agree that it should be checked only for either one of m1, m2 or m3. 
I think I took your comment in the  wrong sense :P 
I'll make the changes now. Sorry for taking so much time for a small issue.